docs/usage/key-concepts/how-it-works.md

```diff
@@ -0,0 +1,152 @@
+---
+title: How it works ?
+description: Learn all about Renovate's workflow
+---
+
+# Introduction
+
+Renovate works by searching for all the dependencies in your project and then look for updates for each.
+
+To be compatible with the variety of dependency-naming and versioning conventions, Renovate has defined modules for each known convention, and you can define your own too.
```

Comment:
```suggestion
Because Renovate needs to support a lot of dependency naming and versioning conventions, it has modules for each known convention.
You can define your own modules, if you want.
```